src/java.base/share/classes/jdk/internal/io/JdkConsoleImpl.java

```diff
@@ -56,6 +56,40 @@ public Reader reader() {
         return reader;
     }
 
+    @Override
+    public JdkConsole println(Object obj) {
+        pw.println(obj);
+        // pw is constructed with autoFlush == true
+        return this;
+    }
+
+    @Override
+    public JdkConsole print(Object obj) {
+        pw.print(obj);
+        // pw is constructed with autoFlush == true
+        return this;
```

Comment:
I'm not sure whether autoflush is sufficient here. I think flushing might be done by print() only when what's printed actually contains a newline. If someone calls `print("Prompt: ")` then autoflush might not flush in this case, but we want flushing.

Reply:
L69 is actually a bug; will fix it.

Reply:
I'm not sure why GitHub hasn't marked this as "Outdated", but it was fixed in f710373, which I pushed 5 minutes ago.